src/main/java/io/openliberty/tools/intellij/actions/RunLibertyDevTask.java

```diff
@@ -68,13 +68,14 @@ private static void getLibertyTree(@NotNull AnActionEvent e, Project project, bo
                     if (view instanceof Tree) {
                         libertyTree = (Tree) view;
                         TreePath[] selectionPaths = libertyTree.getSelectionPaths();
-                        // when only one child node is selected, enable this action
                         if (selectionPaths != null && selectionPaths.length == 1) {
                             String lastPathComponent = selectionPaths[0].getLastPathComponent().toString();
                             if (Constants.FULL_ACTIONS_MAP.containsKey(lastPathComponent)) {
                                 if (isUpdate) {
                                     e.getPresentation().setEnabled(true);
+                                    // when only one child node is selected, enable this action
```

Comment:
It is standard for comments to apply to the lines they are on or the lines below. It may be misleading to have this comment after the line it is meant to be about.